src/44_Config_Start_Tasks.md

```diff
@@ -14,48 +14,30 @@ tasks:
 ```
 
 You can have multiple tasks, which are opened on separated terminals.
-Let's go through the supported properties:
 
-## command
+## Defining Commands
 
-The mandatory `command` is executed in a new terminal running in a bash. YAML allows to write string values without quotes and
-even has a nice syntax for multi-line strings which comes in handy if you chain mutliple commands using `&&`.
+The `command` property is used to specifiy the command that shall be started on ever start of a workspace.
+It is a bash command and doesn't need to terminate. For instance, you could start a web server or a database.
 
-Here is an example for a multi-line task:
+The `init` property can be used to specify shell commands that should only be executed on after a workspace was freshly cloned and needs to be initialized somehow.
```

Comment:
The description of `before` is missing - I think it would make sense to list the phases the workspace goes through, i.e. a list of cases:
- freshly started: before, init, command
- restarted/snapshot: before?, command